cypress/integration/api.spec.js

```diff
@@ -18,8 +18,11 @@ metro_code:	0
 
 describe('User Ip geolocation service tests', () => {
   it('Smoke - User Ip geolocation service (API)', () => {
-   geoService
+    cy.allure().tms('Smoke API', 'AUT-12');
+    cy.allure().startStep(`Check geoService`)
```

Comment:
cy.allure().startStep(`Check geoService`) and  cy.allure().endStep(); must be only inside method in checkGeoService()